File: c/src/menus.c
// src/menus.c


#include <stdio.h>
#include "menus.h"

void menuPrincipal() {
    printf("\n===== BIBLIOTECA =====\n");
    printf("1. Ver catálogo\n");
    printf("0. Sair\n");
    printf("======================\n");
}

int lerOpcao() {
    int opcao;
    printf("Escolha uma opção: ");
    scanf("%d", &opcao);
    return opcao;
}

File: java/src/ui/Menus.java
// src/interface/Menus.java


package ui;

public class Menus {

    public static void menuPrincipal() {
        System.out.println("╔════════════════════════════════════╗");
        System.out.println("║        Biblioteca (Java)           ║");
        System.out.println("╠════════════════════════════════════╣");
        System.out.println("║ 1. Ver catálogo                    ║");
        System.out.println("║ 0. Sair                            ║");
        System.out.println("╚════════════════════════════════════╝");
    }

    public static void menuEscolhaLivro(int total) {
        System.out.println("1 - " + total + ". Escolher livro");
        System.out.println("0. Voltar");
    }

    public static void menuLivroDisponivel() {
        System.out.println("1. Requisitar");
        System.out.println("2. Adicionar avaliação");
        System.out.println("0. Voltar");
    }

    public static void menuLivroEmprestado() {
        System.out.println("1. Devolver");
        System.out.println("2. Adicionar avaliação");
        System.out.println("0. Voltar");
    }
}
